service/src/main/java/org/cbioportal/service/impl/ClinicalAttributeServiceImpl.java

```diff
@@ -52,15 +59,19 @@ public ClinicalAttribute getClinicalAttribute(String studyId, String clinicalAtt
     @PreAuthorize("hasPermission(#studyId, 'CancerStudy', 'read')")
     public List<ClinicalAttribute> getAllClinicalAttributesInStudy(String studyId, String projection, Integer pageSize,
                                                                    Integer pageNumber, String sortBy,
-                                                                   String direction) {
+                                                                   String direction) throws StudyNotFoundException {
+
+        studyService.getStudy(studyId);
```

Comment:
Yes, we don't need the return value, as long as doesn't throw an exception, we continue.